jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/registration.jsp

```diff
@@ -32,7 +32,7 @@
 
 <div class="container form-login-related registration-page">
   <form:form id="form" name="form" action='${pageContext.request.contextPath}/user/new'
-             modelAttribute="newUser" method="POST" class='form-vertical'>
+             modelAttribute="newUser" method="POST" class='form-vertical' tabindex="-1">
```

Comment:
That's very weird.. I can reproduce the issue on QA with my FF 41.0.2. Maybe there is a difference between FFs on different platform though that's still weird. 

So given the bug is reproduced on others' machines, do you think that current fix is applicable?